app/helpers/middleware_server_helper/textual_summary.rb

```diff
@@ -0,0 +1,55 @@
+module MiddlewareServerHelper::TextualSummary
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    @prop_hash = eval(@record.properties)
+
+    %i(name host bind_addr product version)
+  end
+
+  def textual_group_relationships
+    # Order of items should be from parent to child
+    %i(ems middleware_deployments)
+    # items.concat(%i(middleware_servers))
+    # items
+  end
+
+  def textual_group_smart_management
+    %i(tags)
+  end
+
+  #
+  # Items
+  #
+
+  def textual_name
+    @record.name
+  end
+
+  def textual_host
+    @record.host
+  end
+
+  def textual_bind_addr
```

Comment:
This new column needs a dictionary entry because it appears as "Bind addr" in the UI, which should probably be "Bind Address" or something.